packages/youtube_player_iframe_web/lib/src/web_youtube_player_iframe_controller.dart

```diff
@@ -217,6 +216,15 @@ class YoutubePlayerIframeWeb extends PlatformWebViewWidget {
   }
 }
 
+String handleJsMessage(Object? jsMessage) {
+  return switch (jsMessage) {
+    String message => message,
+    Map map => jsonEncode(map),
+    Object? data => throw Exception(
```

Comment:
Let's fail silently

Reply:
Implemented in [this commit](https://github.com/sarbagyastha/youtube_player_flutter/pull/975/commits/234f46b50f63fe593c28d2f582275d7abb84da35)